src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -5,6 +5,8 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System.Globalization;
```

Comment:
We don't do this in our ref files.  Instead use the namespace-qualified name on every required usage.